src/js/pie.js

```diff
@@ -158,15 +167,37 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         that.texts.text(function (d) {
             return floe.d3ViewComponent.getTemplatedDisplayValue(totalValue, percentageDigits, sliceTextDisplayTemplate, d);
         });
+    
+        that.paths.each(function (d) {
+            that.trackD3BoundElement(d.data.id, this);
+        });
+
+        that.texts.each(function (d) {
+            that.trackD3BoundElement(d.data.id, this);
+        });
 
         that.texts.transition().duration(animationDuration).attr("transform", that.textTransform);
 
     };
 
     floe.chartAuthoring.pieChart.pie.removeSlices = function (that) {
-        that.paths.exit().remove();
+        var removedSlices = that.paths.exit();
```

Comment:
`tomorrow && tomorrow && tomorrow`?

`recordedTime.syllables.last()`?

That's all I got :(